packages/libs/eda/src/lib/core/components/visualizations/implementations/VolcanoPlotVisualization.tsx

```diff
@@ -415,6 +416,12 @@ function VolcanoPlotViz(props: VisualizationProps<Options>) {
         ]
       : [];
 
+  // Record any floors for the p value and adjusted p value sent to us from the backend.
+  const statisticsFloors: StatisticsFloors = {
+    pValueFloor: (data.value && Number(data.value.pValueFloor)) ?? 0,
```

Comment:
Similarly, if our default `DefaultStatisticsFloors` object assigns `1e-200` for `pValueFloor`, then should this fallback value not also be `1e-200` (and be derived from that same object)?